nexus/db-queries/src/db/datastore/dataset.rs

```diff
@@ -180,6 +182,56 @@ impl DataStore {
 
         Ok(all_datasets)
     }
+
+    pub async fn dataset_on_in_service_physical_disk(
+        &self,
+        // opctx: &OpContext,
+        dataset_id: Uuid,
+    ) -> LookupResult<bool> {
+        //let conn = self.pool_connection_authorized(opctx).await?;
```

Comment:
Presumably you want to restore this before merging. What caused this change to be needed in the first place?

Reply:
Ok, that makes sense.